js/imgur.js

```diff
@@ -3,7 +3,7 @@
 
 var weechat = angular.module('weechat');
 
-weechat.factory('imgur', ['$rootScope', function($rootScope) {
+weechat.factory('imgur', 'settings', ['$rootScope', function($rootScope) {
```

Comment:
Because of this I'm getting `angular.js:15536 Error: [ng:areq] http://errors.angularjs.org/1.7.5/ng/areq?p0=fn&p1=not%20a%20function%2C%20got%20string`

See https://stackoverflow.com/questions/21673404/error-ngareq-from-angular-controller

You should not add the 'settings' there.

Reply:
I'm testing that.

Reply:
Do 

`weechat.factory('imgur', ['$rootScope', 'settings', function($rootScope, settings) {`

Reply:
Now fixed.